### DISCUSSION TITLE
Classic Menu remnants - remove

### DISCUSSION THREAD

**Discussion by EnSee** [Open-Shell/Open-Shell-Menu#1848] [General]
Hi

I'm running a clean install of open shell and windows 11 latest.
Mistakenly installed classic menu initially.
Have done a repair install and removed classic menu
However, there appears to be a remnant of that code still active
I still get a white band menu with standard drop down menus where classic menu used to be.
Have tried repeating this process a few times

This is pretty annoying since I normally run in dark mode so it's very obvious

Can anyone help with registry entries or similar to check for and remove to eliminate this issue?

Cheers

**Comment by ge0rdi** [comment]
I think you may try this https://github.com/Open-Shell/Open-Shell-Menu/issues/326#issuecomment-813086363.

**Comment by EnSee** [comment]
Thanks

This seems the right issue and fix
However, the relevant key seems to be present already, and the setting
Unless I can't read reg entries properly

See screenshot
![2024-02-29 11_16_39-Greenshot](https://github.com/Open-Shell/Open-Shell-Menu/assets/49475521/85e36f40-c7ea-436a-8e0f-b1ffb91ba77b)

**Reply by ge0rdi** [reply]
You need to set `Locked` to `1`.

**Reply by EnSee** [reply]
Thanks

It was set to 1

Though now it seems to be working :-/

**Reply by EnSee** [reply]
Thanks again for this

Didn't fix the issue. 
See screenshots

Also, the folder icons size is very small, seems linked to OS also
![2024-03-13 09_13_30-Greenshot](https://github.com/Open-Shell/Open-Shell-Menu/assets/49475521/efd4416e-2de7-439e-955e-68758a146a19)
![2024-03-13 09_13_02-Greenshot](https://github.com/Open-Shell/Open-Shell-Menu/assets/49475521/f3bdb1ff-b531-4746-8e4b-1c7d8240c0c0)

**Reply by ge0rdi** [reply]
> Didn't fix the issue.
> See screenshots

I guess you are referring to the white menu bar (with File, Edit, View, Tools).
I have no clue where that is coming from. `Classic Explorer` doesn't do anything like that :(

> Also, the folder icons size is very small, seems linked to OS also

Don't think this is related to `Open-Shell` as it doesn't do anything like that.

Could it be you have some other 3rd party customization software?
